workers/scanner.go

```diff
@@ -149,17 +149,17 @@ func (s *Scanner) threadedScanAndPin() {
 			}
 		}
 
-		s.staticLogger.Tracef("Start scanning.")
+		s.staticLogger.Tracef("Start scanning")
```

Comment:
That's a great suggestion and I would have certainly gone for it if we had a way to tune the log level per some logical group. But since we don't, it will just result in more logspam. I've tried to hit a useful middle ground of logging just enough to know the execution path. This is basically the logging I used while developing and testing this (with some adjustments, obviously).

Reply:
Resolving this because it doesn't really matter but I just meant adding 

```go
s.staticLogger.Tracef("Start scanning")
s.staticLogger.Tracef("1/3 managedRefreshDryRun...")
s.managedRefreshDryRun()
s.staticLogger.Tracef("2/3...")
...
```